### ISSUE TITLE
What determines what is displayed in a table cell?

### ISSUE THREAD

**Issue by simPod** [TrestleAdmin/trestle#363]
Consider this model and table setup for movies.

```ruby
class Director < ApplicationRecord
  belongs_to :movies

  validates :birth_year
  validates :name
end

class Movie < ApplicationRecord
  belongs_to :director

  validates :name
end

Trestle.resource(:movies) do
   table do
     column :name
     column :director
```

What determines what content/column value is displayed in `director` column?

**Comment by spohlenz** [comment]
TLDR: Set/append to `config.display_methods` to determine the methods to call on a model instance when rendering a column value. Defaults to `[:display_name, :full_name, :name, :title, :username, :login, :email]`

There are a few layers here (each of which I need to document better). When a table renders the content for a cell, it calls [`format_value`](https://github.com/TrestleAdmin/trestle/blob/master/app/helpers/trestle/format_helper.rb) on the column's value (either the result of calling the column name as a method on the instance, or the result of the block).

If the column value is a model instance, it will call the [`display`](https://github.com/TrestleAdmin/trestle/blob/master/app/helpers/trestle/display_helper.rb) helper with the instance, which delegates to the [`Trestle::Display`](https://github.com/TrestleAdmin/trestle/blob/master/lib/trestle/display.rb) class. This class simply iterates over the methods listed in `Trestle.config.display_methods`, which defaults to:

```ruby
[:display_name, :full_name, :name, :title, :username, :login, :email]
```
